tessera-dist/src/main/java/com/quorum/tessera/launcher/Launcher.java

```diff
@@ -127,10 +150,51 @@ public void launchServer(Config config) throws Exception {
 
   public abstract void launchServer(Config config) throws Exception;
 
-  public static Launcher create(boolean isRecoveryMode) {
+  public static Launcher create(final boolean isRecoveryMode) {
     if (isRecoveryMode) {
       return Launcher.RECOVERY;
     }
     return Launcher.NORMAL;
   }
+
```

Comment:
We might want to avoid adding logics into the Launcher as these are hidden and not covered by unit tests